### ISSUE TITLE
Investigate getting rid of PyTorch Lightning

### ISSUE THREAD

**Issue by sdatkinson** [sdatkinson/neural-amp-modeler#326]
Working on #321, I'm somewhat worried that the cause of the environment's finickiness is PyTorch Lightning and the dependencies it pulls in (torchmetrics, transformers(?!), HuggingFace hub).  It may be worth it to re-roll the trainer first-party to stop depending on them.

Key questions: 
- [ ]  Is this really the cause of the difficulties?
- [ ] Can this be done without affecting performance?

I'm sure I can replicate (or really, improve on) the framework for training (mostly, the inclusion of optimizers and loss functions inside the model's class definition bothers me).

**Comment by sdatkinson** [comment]
Not going to be looking at this right now.
